src/main/java/seedu/address/logic/parser/MasterDeckParser.java

```diff
@@ -208,6 +208,23 @@ public Command parseCommandInMainSelectedMode(String userInput) throws ParseExce
         case FindCardsCommand.COMMAND_WORD:
             return new FindCardsCommandParser().parse(arguments);
 
+        // Other Commands
```

Comment:
just curious why did you shift Other Commands to be in front of Review-related Commands?

Reply:
No reason actually. I'm going to shift it to be after Review-related commands now :>